Jenkinsfile

```diff
@@ -15,36 +15,37 @@ elifePipeline {
             stage 'Project tests', {
                 // ensure a clean starting state
                 sh "docker-compose down -v"
+
                 sh "docker-compose run ci ./project_tests.sh"
+                // it is not yet possible to retrieve a JUnit XML log to archive as a test artifact:
+                // - the `xunit` formatter mangles the XML outputting also debug statements between tags
+                // - the `xunit-file` formatter, which is an external plugin, doesn't seem to work with gulp-mocha-phantomjs
+            }
+
+            stage 'Smoke tests', {
+                sh "docker-compose run ci ./smoke_tests.sh ui http"
+
                 // preserve environment to allow investigation if build fails, clean up otherwise
                 sh "docker-compose down -v"
             }
-        },
-        'containers--medium'
-    )
-
-    stage 'Smoke tests', {
-        lock('pattern-library--ci') {
-            builderDeployRevision 'pattern-library--ci', commit
-            builderSmokeTests 'pattern-library--ci', '/srv/pattern-library'
```

Comment:
This is the cause of losing the separate `smoke_tests` PR status?

Reply:
Yes, because the new smoke tests are implemented manually to show the docker commands used. I'll wrap them into the Github commit status "context manager".